pages/docs/concepts/producer.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Producer
+weight: 3
+---
+
+## What is a Producer? 
+
+An event indicates a state change or update that is triggered by a user's/device's action; this could be something like an item being placed in a shopping cart on an e-commerce website, clicking the subscribe button on a YouTube channel, or a temperature change in a sensor. A producer is a type of application that senses that state change and then publishes that event as a message.
+
+## Why do we need Producers? 
+
+One of the core concepts of event-driven architecture is the publish/subscribe communication model. Producers are publishers in this model and the first logical layer responsible for publishing messages to the broker. This way, any consumer subscribed to that event can handle the published messages.
```

Comment:
```suggestion
One of the core concepts of event-driven architecture is the publish/subscribe communication model. Producers are publishers in this model and the first logical layer responsible for publishing to the broker messages that others can subscribe to receive.
```

maybe let us not mention subscribers here to much? to not complicate message. Especially that producer do not care who and if any subscriber is awaiting the message.